projects/demo-integrations/cypress/tests/demo/demo.spec.ts

```diff
@@ -2,15 +2,31 @@ import {DEMO_PATHS} from '../demo-paths';
 import {EXAMPLE_ID} from '../shared.entities';
 import {excluded} from './exclusions';
 
+// not taiga ui icons
+const EXTERNAL_ICONS = ['web-api.svg'];
+
+const stubIcons = (icons: string[]): void => {
+    icons.forEach(iconName => {
+        cy.intercept(
+            {
+                method: 'GET',
+                url: new RegExp(`.*${iconName}`),
+            },
+            {fixture: `icons-stubs/${iconName}`},
+        );
+    });
+};
+
 describe('Demo', () => {
     beforeEach(() => {
         cy.viewport(1280, 720);
     });
 
     DEMO_PATHS.forEach(path => {
         it(`${path}`, () => {
+            stubIcons(EXTERNAL_ICONS);
```

Comment:
Can this function be used one time? in `beforeAll` for example

Reply:
According to [cypress docs](https://docs.cypress.io/api/commands/intercept):

> All intercepts are automatically cleared before every test

If we place this function in `beforeAll`, the stub will work for the first test only :(

Reply:
got it :ok_man: